{% extends "bootstrap/base.html" %}
{% import "bootstrap/wtf.html" as wtf %}

{% block title %}
Help
{% endblock %}

{% block styles %}
{{super()}}
<link rel="stylesheet"
      href="{{url_for('.static', filename='styles.css')}}">
{% endblock %}

{% block content %}
<nav class="navbar navbar-inverse navbar-fixed-top">
      <div class="container">
        <div class="navbar-header">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand" href="{{ url_for('index') }}">AmphiProt</a>
        </div>
        <div id="navbar" class="collapse navbar-collapse">
          <ul class="nav navbar-nav">
            <li><a href="{{ url_for('index') }}">Home</a></li>
            <li><a href="{{ url_for('loging') }}">Login</a></li>
            <li><a href="{{ url_for('register') }}">Sign Up</a></li>
            <li class="active"><a href="{{ url_for('help') }}">Help</a></li>
          </ul>
        </div><!--/.nav-collapse -->
      </div>
</nav>
<div class="container">
        <h1 class="help">Help</h1>
        <h2 class="help">About us</h2>
        <p class="help">
        Welcome to the web application <a href="{{ url_for('index') }}"><i>AmphiProt</i></a>. This app has been developed by Álvaro Ciudad, Silvia González, Sergio Suárez and Eric Toro and aims to retrieve different analysis on protein sequences.
        </p>
        <h2 class="help">Context and relevance of the application</h2>
        <p class="help">
        Most proteins present amphipathic structures, with one hydrophilic face with affinity to water and one hydrophobic face which repels it. Particularly, canonical alpha helices with about 3.6 amino acids per turn and canonical beta sheets consisting of 2 amino acids per turn are the most common amphipathic structures in these macromolecules. The search of these structural features has awakened especial interest, since they are usually found on surface regions of the protein, which can participate in active centers or binding domains that could be potential
antigenic epitopes for the creation of new promising vaccines as well as inhibitory molecules of therapeutic interest.
        </p>
        <p class="help">
        In this application, a methodology based on Fourier transform of hydrophobicities is implemented in order to identify periodic properties on query amino acid sequences. This periodicity gives an insight on the secondary structure subjacent to the protein region of interest.
        </p>
        <p class="help">
        Users can submit their sequence of interest in various formats. The application then displays both an amphipathic and a hydrophobicity plot buy default, although additional analysis on homolog sequences (BLASTP) and isoelectric point of the sequence can also be chosen by the user.
        </p>
        <h2 class="help">How can I use the application?</h2>
        <p class="help">
        When using this app, two access options are available:
        <ul class="help">
        	<li class="help"><b>Registered user</b>: By registering on the <a href="{{ url_for('register') }}"><i>Register</i></a> page, you can perform your analysis and have available the last 10 sets of results that you have obtained accessing your workspace every time you <a href="{{ url_for('loging') }}"><i>log in</i></a>.</li>
        	<li class="help"><b>Non-registered user</b>: Registration is not compulsory to use the app. However, once you close the session in your browser, all the data that you have obtained on your query will be lost.</li>
        </ul>
        </p>
        <h2 class="help">How does it work?</h2>
        <p class="help">
        In the <a href="{{ url_for('index') }}"><i>Home</i></a> page you will find a form with various fields:
        <ul class="help">
        	<li class="help"><b>PDB_id</b>: Here, users can introduce the PDB code (4 uppercase alphanumeric characters) of the protein that they want to analyse.</li>
        	<li class="help"><b>UniProt_id</b>: Here, users can introduce the UniProt code (between 4 and 16 uppercase alphanumeric characters) of the protein that they want to analyse.</li>
        	<li class="help"><b>Sequence</b>: Field in which the query amino acid sequence can be wirtten in FASTA format.</li>
        	<li class="help"><b>File</b>: The user can also upload a file in FASTA format containing the query sequence that they want to analyise.</li>
        	<li class="help"><b>Table</b>: As the analysis performed by this app on amphipathic regions require the use of a normalized hydrophobicity scale, users can choose which table of scores they want to apply to the study of their protein from Chothia, Janin, Tanford, Wimley, Eisenberg, Kyte & Doolittle, von Heijne-Blomberg and Wolfenden.</li>
        	<li class="help"><b>BLAST</b>: users also have the option of running a BLASTP analysis on their protein sequence of interest when launching the analysis.</li>
        	<li class="help"><b>Isoelectric point</b>: Selecting this option, the computation of the isoelectric point of the sequence will also be retrieved to the user.</li>
        </ul> 
        </p>
</div><!--/.nav-collapse -->
     
     
     
     <!-- /container -->
{% endblock %}
